fp-nbloniarz-Natalie-Bloniarz-AnaghaLate-Anagha-Late

nbloniarz opened the pull request.

nbloniarz committed:
Create proposal.md

charlieroberts commented:
This sounds like a great project! Do you know how you'll be doing the sounds / sequencing yet? I'm happy to make some recommendations if you need them...

nbloniarz committed:
Mostly Working Piano Playback

It looks bad and has some interesting interactions

nbloniarz committed:
Started to outline database interaction

its at the end of the file

nbloniarz committed:
VS Updates

TODO: Integrate Metronome, Variable Recording Length, Add Drums, Countdown To Start Recording

nbloniarz committed:
Updated Piano Visuals

nbloniarz committed:
Finalized Piano

TODO: Drums, Metronome, And Whole Playback

nbloniarz committed:
Setup for Database Integration

Still Needs Drums

nbloniarz committed:
Drum Track Setup

nbloniarz committed:
Adding Guitar Files -To Be Removed in a Sec

nbloniarz committed:
Removed GuitarFor Master

nbloniarz committed:
Fixed playback -Now fixing asthetic

nbloniarz committed:
VISUALS DONE

nbloniarz committed:
Added Pub VS Pri

gameNoSave is the public Version
game is the logged in Version

Commit:
🚀👻 Checkpoint

Commit:
🐏👮 Checkpoint
./index.html:6273880/251

Commit:
🐦⛸ Checkpoint
./index.html:6273880/2478

Commit:
⛵️💡 Checkpoint
./index.html:6273880/344

Commit:
⛵️🍿 Checkpoint
./server.improved.js:6273880/7957
./js/scripts.js:6273880/776

Commit:
🎈👶 Checkpoint
./package.json:6273880/2424
./index.html:6273880/1107
./server.improved.js:6273880/11270
./js/scripts.js:6273880/2190

Commit:
💏👁 Checkpoint
./index.html:6273880/282

Commit:
⚓️📮 Checkpoint
./index.html:6273880/2339
./server.improved.js:6273880/258

Commit:
🚈🍪 Checkpoint
./js/scripts.js:6273880/776

Commit:
👹👼 Checkpoint
./package.json:6273880/604
./index.html:6273880/1849

Commit:
🙌💁 Checkpoint
./index.html:6273880/4567
./server.improved.js:6273880/224

Commit:
☔️👄 Checkpoint
./index.html:6273880/290
./server.improved.js:6273880/18
./Temp.css:6273880/141

Commit:
🏛🌧 Checkpoint
./Temp.css:6273880/164
./index.html:6273880/5

Commit:
🏅💡 Checkpoint
./Temp.css:6273880/12552
./index.html:6273880/2777

Commit:
🍮🛬 Checkpoint
./server.improved.js:6266250/125

Commit:
🚟💿 Checkpoint
./server.improved.js:6266250/6351

Commit:
🏔✊ Checkpoint
./server.improved.js:6266250/891
./index.html:6266250/87
./gameNoSave.html:6266250/10090
./game.html:6266250/10303

Commit:
🛩🍴 Checkpoint
./server.improved.js:6266250/18555
./js/scripts.js:6266250/1036

Commit:
Login Works - Moving Offline

Commit:
🎓👜 Checkpoint
./js/scripts.js:6266250/4093
./index.html:6266250/153
./game.html:6266250/10655
./server.improved.js:6266250/132

Commit:
🦂❣️ Checkpoint
./server.improved.js:6266250/354
./Temp.css copy:6266250/546
./game.html:6266250/329
./index.html:6266250/11

Commit:
👽🚡 Checkpoint
./server.improved.js:6266250/537
./js/scripts.js:6266250/996
./login.html:6266250/169

Commit:
🏠🤖 Checkpoint
./server.improved.js:6266250/12396
./js/scripts.js:6266250/4675

Commit:
💼🕋 Checkpoint
./js/scripts.js:6266250/855
./game.html:6266250/22

Commit:
🌖💼 Checkpoint
./js/scripts.js:6266250/5834
./game.html:6266250/200
./game.js:6266250/462

Commit:
🍏🌵 Checkpoint
./game.js:6266250/1256

Commit:
🏆🐲 Checkpoint
./game.js:6266250/287702
./server.improved.js:6266250/14203

Commit:
Fixed total playback, finishing up interaction

Commit:
🏵🚅 Checkpoint
./game.js:6266250/883
./game.html:6266250/74
./index.html:6266250/133

Commit:
😀👱 Checkpoint
./js/scripts.js:6266250/625
./game.js:6266250/1296
./game.html:6266250/277
./index.html:6266250/10452

Commit:
🌮🐜 Checkpoint
./game.js:6266250/357
./server.improved.js:6266250/256

Commit:
🐒💳 Checkpoint
./server.improved.js:6266250/457

Commit:
💝🏙 Checkpoint
./server.improved.js:6266250/177

Commit:
🛬💋 Checkpoint
./game.js:6266250/49
./game.html:6266250/137
./Temp.css:6266250/748

Commit:
🏔🐵 Checkpoint
./game.html:6266250/146
./Temp.css:6266250/4
./game.js:6266250/194

Commit:
🌿🙎 Checkpoint
./Temp.css:6266250/539
./game.js:6266250/16
./game.html:6266250/119

Commit:
👄🛎 Checkpoint
./game.html:6266250/1251
./Temp.css:6266250/38
./login.html:6266250/537

Commit:
🚣💖 Checkpoint
./game.html:6266250/3731
./game.js:6266250/16365

Commit:
📫🌯 Checkpoint
./game.js:6266250/137608
./game.html:6266250/301

Commit:
🚞🍪 Checkpoint
./game.js:6266250/1374
./index.html:6266250/672
./game.html:6266250/121
./server.improved.js:6266250/184

Commit:
Its Done?

Commit:
📟🚴 Checkpoint
./server.improved.js:6266250/14222
./index.html:6266250/44
./README.md:6266250/1507

Commit:
Updated Readme

AnaghaLate committed:
Update README.md

Commit:
🍢⛈ Checkpoint
./README.md:6266250/12
./game.js:6266250/139470

Commit:
final Push

Commit:
Fixed Readme failure to merge

Commit:
Fixed Readme failure to merge
